release-notes/2018.md

```diff
@@ -10,14 +10,6 @@ description: Release Notes for 2018
 
 * Fixed a bug where the `deviceToken` conversion was incorrect in some situations.
 
```

Comment:
Why we are keeping the other  iOS releases on this  release page?
I saw you remove the iOS  releases for 2019.

Reply:
I only cleaned this up for experience cloud extensions, just to keep the PR cleaner. We can remove those as part of the foundation extensions update.